check_stability.py

```diff
@@ -424,10 +424,22 @@ def write_inconsistent(inconsistent, iterations):
     table(["Test", "Subtest", "Results"], strings, logger.error)
 
 
-def write_results(results, iterations):
+def write_results(results, iterations, comment_pr):
     logger.info("## All results ##\n")
     for test, test_results in results.iteritems():
-        logger.info("### %s ###" % test)
+        baseurl = "http://w3c-test.org/submissions"
+        if test.split('.')[-2] == "https":
```

Comment:
Technically I think you want `"https" in os.path.splitext(test)[0].split(".")[1:]` or something.